contracts/SimpleManager.sol

```diff
@@ -315,6 +309,70 @@ contract SimpleManager is OwnableUpgradeable {
         return _operators.contains(operator_);
     }
 
+    // solhint-disable-next-line function-max-lines
+    function _checkPoolsPrices(
+        uint256 mintsLength_,
+        VaultInfo memory vaultInfo_,
+        Rebalance calldata rebalanceParams_,
+        address token0_,
+        address token1_,
+        uint8 token0Decimals_,
+        uint8 token1Decimals_
+    ) internal view {
+        if (mintsLength_ == 0) return;
+
+        uint24[] memory checked = new uint24[](mintsLength_);
+        uint256 oraclePrice = vaultInfo_.oracle.getPrice0();
+        uint256 increment;
+        for (uint256 i; i < mintsLength_; ++i) {
+            if (
+                _includes(
+                    rebalanceParams_.mints[i].range.feeTier,
+                    checked,
+                    increment
+                )
+            ) continue;
+
+            IUniswapV3Pool pool = IUniswapV3Pool(
+                _getPool(
+                    token0_,
+                    token1_,
+                    rebalanceParams_.mints[i].range.feeTier
+                )
+            );
+
+            uint256 sqrtPriceX96;
+
+            (sqrtPriceX96, , , , , , ) = pool.slot0();
+
+            uint256 poolPrice;
+
+            if (sqrtPriceX96 <= type(uint128).max) {
+                poolPrice = FullMath.mulDiv(
+                    sqrtPriceX96 * sqrtPriceX96,
+                    10 ** token0Decimals_,
+                    2 ** 192
```

Comment:
Consider changing to `1 << 192` for consistency with `1 << 128`.

Reply:
Yes will make the change.